src/encoded/static/components/browse/columnExtensionMap.js

```diff
@@ -128,7 +128,7 @@ export const columnExtensionMap = _.extend({}, basicColumnExtensionMap, {
             if (!number_of_experiments){
                 number_of_experiments = 0;
             }
-            return <span key="val">{ number_of_experiments }</span>;
+            return <span className="value">{ number_of_experiments }</span>;
```

Comment:
Centers the text content. Forgot to add likely after Bootstrap upgrade & related edits.